packages/Sortable/README.md

```diff
@@ -0,0 +1 @@
+Sortable items are rad.
```

Comment:
can you add a simple example of how to use the component, this will appear on npm page.

Reply:
Or at least a bad "sorta" pun.

Reply:
🤦‍♂️ such a squandered opportunity...